packages/@coorpacademy-nova-icons/third-party/nova-composition.iconjar/icons/validate.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 170 170" style="enable-background:new 0 0 170 170;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill-rule:evenodd;clip-rule:evenodd;fill:#3EC483;}
+	.st1{fill:#FFFFFF;}
```

Comment:
can you pleas remove the css and put it on attribute please ? because in native we need fill attributes (and fill-xxx too) to be set in tags (g, path, etc)